platform/rct/hal/LED_stm32/led_device_stm32.c

```diff
@@ -43,9 +43,9 @@ static struct led_data {
         uint16_t mask;
         bool level;
 } leds[] = {
-        {LED_ERROR,     GPIO_Pin_2, 0},
+        {LED_ERROR,     GPIO_Pin_0, 0},
         {LED_GPS,       GPIO_Pin_1, 0},
-        {LED_TELEMETRY, GPIO_Pin_0, 0},
+        {LED_TELEMETRY, GPIO_Pin_2, 0},
```

Comment:
Yes, the previous boards were actually incorrect. I moved it around and also chose new colors.